packages/atlas/src/components/_video/VideoTilePublisher/VideoTilePublisher.tsx

```diff
@@ -121,10 +119,13 @@ export const VideoTilePublisher: FC<VideoTilePublisherProps> = memo(
     const hasThumbnailUploadFailed =
       (video?.thumbnailPhoto &&
         !video.thumbnailPhoto.isAccepted &&
-        uploadThumbnailStatus?.lastStatus !== 'completed') ||
+        uploadThumbnailStatus?.lastStatus !== 'completed' &&
+        !video.ytVideoId) ||
```

Comment:
![image](https://user-images.githubusercontent.com/51168865/223699172-2670ac19-87e8-4cea-9446-cbcc9834adb9.png)

So this is how it should work? For some time user won't be able to see his thumbnail at all? This is far from perfect to me. 

Also, what will happen if the user decides to change the thumbnail on the atlas side, he'll update the new image and the upload fails? He won't be able to know what's happening, because `hasThumbnailUploadFailed` won't be false if the ytVideoId is set.

Reply:
resolved on the call